src/server/api/routers/auth.ts

```diff
@@ -46,7 +46,85 @@ export const authRouter = createTRPCRouter({
       });
       return res;
     }),
-
+  getAllParticipant: publicProcedure.query(async({ctx})=>{
+    const participants = await ctx.db.query.user.findMany({
+      where : eq(user.role,"participant"),
+      columns:{
+        aka: true,
+        bio: true,
+        email: true,
+        firstName: true,
+        gender: true,
+        role: true,
+        lastName: true,
+        profileImageURL: true,
+      }
+    });
+    return participants
+  })
```

Comment:
we should separate "by fiter" and "no filter" into 2 function, both of them are the same thing, we can just passing undefined or null to state that we need all without filtering